.github/workflows/imageupdate.yml

```diff
@@ -0,0 +1,30 @@
+name: Base image auto update
+on:
+  schedule:
+    - cron: "5 0 * * *"
```

Comment:
Is this meant to run once a day at 00:05 or every 5 minutes ?

Reply:
every day at 00:05